### PR TITLE
[gui] Add listitem.status

### PR DESCRIPTION
**Pull request by phil65**
ListItem.status (-->m_strStatus)  seems to only be partly implemented. This makes it actually a usable InfoLabel.

@FernetMenta that VIDEOPLAYER_CONTENT hack is probably some leftover and not needed anymore? @anaconda mentioned that it was abused by mythtv some time ago.    

note: i was lazy in GUIInfoLabels.h and didnt reshuffle the Ids, instead I just appended the new one at the bottom.

### CODE REVIEW

File: xbmc/interfaces/json-rpc/VideoLibrary.cpp [hunk 1]
```diff
@@ -1113,6 +1113,8 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
     details.SetTrailer(parameterObject["trailer"].asString());
   if (ParameterNotNull(parameterObject, "tagline"))
     details.SetTagLine(parameterObject["tagline"].asString());
+  if (ParameterNotNull(parameterObject, "status"))
+    details.SetTagLine(parameterObject["status"].asString());
```
Comment (phil65) [inline xbmc/interfaces/json-rpc/VideoLibrary.cpp]: oops, will fix this.

**Comment by FernetMenta** [comment]
@phil65 what VIDEOPLAYER_CONTENT hack? can you give me a pointer to the code?

**Comment by phil65** [comment]
@FernetMenta 	https://github.com/xbmc/xbmc/pull/9516/files#diff-127e5f2786eb19ac8e0e2f04b3bf8306L7447

**Comment by FernetMenta** [comment]
@phil65 I don't think it is needed anymore. feel free to drop it.

**Comment by phil65** [comment]
jenkins build this please.

**Comment by ronie** [comment]
just wondering, what kind of info does ListItem.Status return?

**Comment by phil65** [comment]
from tmdb scraper for example (i think others have that, too):

> the status field can have a value of returning series (airing right now), in production (airing soon), planned (in development), canceled, or ended.

http://docs.trakt.apiary.io/reference/shows/summary

**Comment by ronie** [comment]
ah, that one. thx!

File: xbmc/interfaces/json-rpc/VideoLibrary.cpp [hunk 2]
```diff
@@ -1113,6 +1113,8 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
     details.SetTrailer(parameterObject["trailer"].asString());
   if (ParameterNotNull(parameterObject, "tagline"))
     details.SetTagLine(parameterObject["tagline"].asString());
+  if (ParameterNotNull(parameterObject, "status"))
+    details.SetStatus(parameterObject["status"].asString());
```
Comment (anaconda) [inline xbmc/interfaces/json-rpc/VideoLibrary.cpp]: This is missing an entry in methods.json:
https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/methods.json#L1481
```json
      { "name": "status", "$ref": "Optional.String" },
```
+ API bump.

**Pull request merged**